README.md

```diff
@@ -1,5 +1,98 @@
 # pydbtools
 
+## Installation
+
+> Requires a pip release above 20.
+
+```bash
+pip install "pydbtools @ git+https://github.com/moj-analytical-services/pydbtools"
+
+## Or install a specific release
+
+pip install "pydbtools @ git+https://github.com/moj-analytical-services/pydbtools@v3.0.0"
+```
+
```

Comment:
I think we should add a quickstart section first. e.g.:

## Quickstart guide

### Read an SQL Athena query into a pandas dataframe

```python
import pydbtools as pydb
df = pydb.read_sql("SELECT * from a_database.table LIMIT 10")
```
### Run a query in Athena using pydbtools

```python
response = pydb.start_query_execution_and_wait("CREATE DATABASE IF NOT EXISTS my_test_database")
```


### Create a temporary table to do further separate SQL queries on later

```python
pydb.create_temp_table("SELECT a_col, count(*) as n FROM a_database.table GROUP BY a_col", table_name="temp_table_1")
df = pydb.read_sql_query("SELECT * FROM __temp__.temp_table_1 WHERE n < 10")
```